### PR TITLE
fix: references to grpc:// protocol removed

### LINKED ISSUE #1520: [TASK] Replace references to grpc:// to https:// or http://
**Overview:**

We've gotten some questions about zeebe-gateway using urls that look like 
`grpc://zeebe-gateway.ingress...`

especially since the web modeler and desktop modeler does not connect to any endpoints prefixed by `grpc://`.

**Actions:**

We should change these references in `helm status` and the Console values print statement to use http or https depending on if TLS is enabled or disabled.

### PR DESCRIPTION
**Pull request by jessesimpson36**
### Which problem does the PR fix?

closes #1520 

### What's in this PR?

References to `grpc://` protocol are replaced with `http://` or `https://`

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [x] In the repo's root dir, run `make go.update-golden-only`.
- [x] There is no other open [pull request](../pulls) for the same update/change.
- [x] Tests for charts are added (if needed).
- [x] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [x] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [x] Did all checks/tests pass in the PR?

### CODE REVIEW

File: charts/camunda-platform/templates/camunda/_helpers.tpl [hunk 1]
```diff
@@ -486,6 +486,14 @@ Zeebe templates.
   {{- end -}}
 {{- end -}}
 
+{{/*
+[camunda-platform] Zeebe Gateway GRPC external URL.
+*/}}
+{{- define "camundaPlatform.zeebeGatewayGRPCExternalURL" }}
+  {{ $proto := ternary "https" "http" .Values.zeebeGateway.ingress.grpc.tls.enabled -}}
+  {{- printf "%s://%s%s" $proto .Values.zeebeGateway.ingress.grpc.host .Values.zeebeGateway.contextPath -}}
```
Comment (npepinpe) [inline charts/camunda-platform/templates/camunda/_helpers.tpl:494]: :x: gRPC does not support context paths unfortunately :( This is likely what's causing the failure in the integration tests.

**Review by npepinpe — APPROVED** [review]
Pre-approving - there's one change needed, removing the context path for gRPC, but otherwise looks good. No need to ping me for another review (unless you'd like, ofc :))

File: charts/camunda-platform/templates/camunda/_helpers.tpl (hunk 1 above)
Comment (jessesimpson36) [reply charts/camunda-platform/templates/camunda/_helpers.tpl]: Dang, you caught me using copy/paste! good catch.

**Comment by jessesimpson36** [comment]
unit tests pass locally. I assume it's failing in this repo due to pull_request being used instead of pull_request_target?

either way, we can ignore that.

**Pull request merged**